lib/page-state/actions.js

```diff
@@ -67,7 +67,7 @@ export function getListData(store, { uri, prefix }) {
       size: 1,
       query: {
         term: {
-          uri // match page uri
+          uri: replaceVersion(uri) // match page uri (removing @published or @scheduled)
```

Comment:
replace version or remove version? the pages index doesn't have any version appended its document keys

Reply:
`replaceVersion` is the name of the function; if passed a single argument it'll remove any version that's applied (or just pass through the uri, if it doesn't have a version)